apps/judicial-system/backend/src/app/modules/case/guards/rolesRules.ts

```diff
@@ -177,6 +177,7 @@ export const prosecutorTransitionRule: RolesRule = {
     CaseTransition.SUBMIT,
     CaseTransition.DELETE,
     CaseTransition.APPEAL,
+    CaseTransition.WITHDRAW_APPEAL,
```

Comment:
Please note that the prosecutor should only be able to withdraw an appeal if the case was appealed by the prosecution. This needs to be reflected in the `canActivate` method below.

Reply:
Good catch! 🙏